src/js/stories/1/dnd.js

```diff
@@ -0,0 +1,15 @@
+//Drag and Drop questions
```

Comment:
For my understanding: Are these questions displayed in the drag and drop screen? And the player has the task to answer these questions with dragging the right thing?

Reply:
Yup, that's right